contracts/src/TroveManager.sol

```diff
@@ -1756,12 +1760,14 @@ contract TroveManager is LiquityBase, ITroveManager, ITroveEvents {
         });
     }
 
+    // This function will revert if there’s a total debt increase and the ratio debt / shares has exceeded the max
     function _updateBatchShares(
         uint256 _troveId,
         address _batchAddress,
         TroveChange memory _troveChange,
         uint256 _batchColl, // without trove change
-        uint256 _batchDebt // entire (with interest, batch fee), but without trove change, nor upfront fee nor redist
+        uint256 _batchDebt, // entire (with interest, batch fee), but without trove change, nor upfront fee nor redist
+        bool _checkBatchSharesRatio
```

Comment:
Ah, so this is not related to whether there's a net debt increase or not - just dictates whether we do the check on the resulting ratio inside the func call

Reply:
Yes, I’ll add a comment to clarify. I can change the name if you have a better suggestion.